npm/postinstall.sh

```diff
@@ -18,7 +19,12 @@ export ETC_DIR=$npm_config_etc
 export SMF_DIR=$npm_config_smfdir
 export VERSION=$npm_package_version
 
-. /lib/sdc/config.sh
+if [[ "$(uname -s)" == "Linux" ]]; then
```

Comment:
I think this might be better as a case statement.

Reply:
Something like this?

```shell
case "$(uname -s)" in
    Linux ) . /usr/triton/bin/config.sh;;
    SmartOS | * ) . /lib/sdc/config.sh;;
esac
```

Reply:
I would do more like this:

```
systype=“$(uname -s)”
case “$systype” in
  Linux) config=/usr/triton/bin/config.sh ;;
  SunOS) config=/lib/sdc/config.sh ;;
  *) printf ‘Unsupported system type: %s\n’ “$systype” ;;
esac
```